### PR TITLE
feat: add pm2 debug servers and add'l debug scripts

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* It can be useful to debug multiple services at once.

This commit:

* Adds a pm2 config that starts 4 of the servers using start-dev-debug
  with unique inspect ports so that they can be debugged all at the
  same time.
* Updates the package.json files for these services so they don't choose
  the same port.
* Documents the ports used for debugging.
* Add's an attach debug for VS Code that lets one choose which process
  to debug.

### CODE REVIEW

File: mysql_servers.json [hunk 1]
```diff
@@ -190,7 +190,8 @@
     },
     {
       "name": "event-broker",
-      "script": "../../_scripts/start_after_mysql.sh",
```
Comment (bbangert) [inline mysql_servers.json]: I had originally copied this from another start, but it doesn't need to wait for mysql so this isn't needed.

File: packages/fxa-content-server/package.json [hunk 2]
```diff
@@ -1,168 +1,168 @@
...
+    "upng-js": "2.1.0",
+    "webpack-dev-middleware": "^3.7.2",
+    "xmlhttprequest": "git://github.com/zaach/node-XMLHttpRequest.git#onerror",
+    "yargs": "^15.0.2"
+  },
+  "engines": {
+    "node": ">=10",
+    "npm": ">=6.4.1"
+  },
+  "readmeFilename": "README.md"
```
Comment (bbangert) [inline packages/fxa-content-server/package.json]: Prettier decided it was time to fix the spacing. 🤷‍♂

File: mysql_servers.json (hunk 1 above)
Comment (dannycoates) [reply mysql_servers.json]: But it does depend on auth-server which waits for mysql. It was crashing for me when it starts before auth.

Comment (bbangert) [reply mysql_servers.json]: Ah, good point! That's why I had that there.

Comment (bbangert) [reply mysql_servers.json]: Reverted.

**Pull request merged**